jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/SwitchesTests.java

```diff
@@ -27,56 +34,149 @@ public void before() {
 
     @Test
     public void functionalityAndCssStyleSwitchTest() {
+        String red = "red";
         final Switch redSwitch = colorSwitches.get(1);
         redSwitch.show();
         redSwitch.is().checked();
-        redSwitch.has().hasLabel();
-        redSwitch.label().has().text("red");
+        redSwitch.has().label();
+        redSwitch.label().has().text(red);
+        redSwitch.has().value(red);
+        redSwitch.has().color(RED.value());
+        redSwitch.has().detailsHidden();
         redSwitch.uncheck();
         redSwitch.is().unchecked();
     }
 
+    @Test
+    public void flatSwitchesTest() {
+        Switch flatSwitch = flatSwitches.get(1);
+        Switch notFlatSwitch = variousAttributesSwitches.get(5);
+        flatSwitch.show();
+        flatSwitch.is().flat();
+        flatSwitch.is().notInset();
```

Comment:
Please delete this and next one check as you have this check in next tests

Reply:
deleted